config/initializers/date_format.rb

```diff
@@ -0,0 +1,5 @@
+Time::DATE_FORMATS[:stepped_cache_key] = lambda { |time|
+  time
+    .change(sec: (time.sec.div(5) * 5))
+    .utc.to_s(:usec)
+}
```

Comment:
Worked out a better way of doing this:

```
Time::DATE_FORMATS[:stepped_cache_key] = lambda { |time| (time.floor - time.to_i % 5).to_s(:usec) }
```

Forgot that Rails 6.1 backports `Time#floor` to earlier rubies.